lib/iris/tests/unit/analysis/regrid/test_RectilinearRegridder.py

```diff
@@ -1263,7 +1263,7 @@ def test_circular_src__masked_missingmask(self):
         # instead of being an array.
         src = self.src
         src.coord('longitude').circular = True
-        src.replace(ma.MaskedArray(src.data), fill_value=src.fill_value)
+        src.data = ma.MaskedArray(src.data, fill_value=src.fill_value)
```

Comment:
In this test `src` was loaded in from a pp file, so the src.fill_value would presumably have been the bmdi value.

However, this test doesn't really care about the fill value so I have just let it use the numpy default fill_value